aioftp/client.py

```diff
@@ -228,11 +232,21 @@ async def command(self, command=None, expected_codes=(), wait_codes=()):
         :param wait_codes: tuple of wait codes or wait code
         :type wait_codes: :py:class:`tuple` of :py:class:`str` or
             :py:class:`str`
+
+        :param censor_after: index after which the line should be censored
+            when logging
+        :type censor_after: :py:class:`None` or :py:class:`int`
         """
         expected_codes = wrap_with_container(expected_codes)
         wait_codes = wrap_with_container(wait_codes)
         if command:
-            logger.info(command)
+            if censor_after:
+                # Censor the user's command
+                logger.info(
+                    command[:censor_after] + "*" * len(command[censor_after:])
+                )
```

Comment:
I think it is better to create "censor"/"non-censor" strings and then concat them without going out of 80 chars.
``` python
command_s = command[:censor_after]
stars = "*" * len(...)
logger.info("%s%s", command_s, stars)
```
or something like this. I'm not a fanat of 80-chars per line right now, but this project historicaly keep this standard.

Reply:
Yeah, the old line at 78 characters was a bit long :sweat_smile:

I've changed it to reflect your suggestion.